### PR TITLE
Dashboard using Inferno

### PR DESCRIPTION
**Pull request by kurkomisi**
Description in the README file.

### CODE REVIEW

**Comment by GitCop** [comment]
Thank you for your contribution! Your commits seem to not adhere to the repository coding standards

 * Commit: bbcc0f80138df551433da60977cccc4233c93552
  * Commits must be prefixed with the package(s) they modify

Please check the [contribution guidelines](https://github.com/ethereum/go-ethereum#contribution) for more details.

---
This message was auto-generated by https://gitcop.com

File: dashboard/README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+## Go Ethereum Dashboard
+### Description
+
+The GED is a data visualizer integrated into geth, intended to collect and visualize useful information of an Ethereum node.
```
Comment (karalabe) [inline dashboard/README.md]: Please no abbreviations :D Just write out "dashboard", no need to get fancy.

**Comment by ligi** [comment]
Great you moved away from react - perhaps close  #15021 then

**Comment by GitCop** [comment]
Thank you for your contribution! Your commits seem to not adhere to the repository coding standards

 * Commit: bbcc0f80138df551433da60977cccc4233c93552
  * Commits must be prefixed with the package(s) they modify

Please check the [contribution guidelines](https://github.com/ethereum/go-ethereum#contribution) for more details.

---
This message was auto-generated by https://gitcop.com

**Pull request closed without merging**